integration-tests/library/src/main/java/com/squareup/anvil/test/AssistedService.kt

```diff
@@ -0,0 +1,16 @@
+package com.squareup.anvil.test
+
+import dagger.assisted.Assisted
+import dagger.assisted.AssistedFactory
+import dagger.assisted.AssistedInject
+
+public class AssistedService @AssistedInject constructor(
+  public val integer: Int,
+  @Assisted public val string: String
```

Comment:
nit: could you use a separate, custom class as the injected type? Something like even a `data class Name(val fullName: String)` might make the example a bit more clear, as injecting a raw `String` isn't a super-typical use case.

Reply:
This really just a tiny integration test and doesn't serve as a sample. An extra class is unnecessary and doesn't make it clearer IMO.